pkg/security/probe/kfilters_test.go

```diff
@@ -96,4 +96,32 @@ func TestIsParentDiscarder(t *testing.T) {
 	if is, _ := isParentPathDiscarder(rs, FileRenameEventType, "rename.old.filename", "/etc/nginx/nginx.conf"); !is {
 		t.Fatal("should be a parent discarder")
 	}
+
+	rs = rules.NewRuleSet(&Model{}, func() eval.Event { return &Event{} }, rules.NewOptsWithParams(SECLConstants, nil))
+	addRuleExpr(t, rs, `unlink.filename =~ "/etc/conf.d/*"`)
+
+	if is, _ := isParentPathDiscarder(rs, FileUnlinkEventType, "unlink.filename", "/etc/sys.d/nginx.conf"); !is {
+		t.Fatal("should be a parent discarder")
+	}
+
+	rs = rules.NewRuleSet(&Model{}, func() eval.Event { return &Event{} }, rules.NewOptsWithParams(SECLConstants, nil))
+	addRuleExpr(t, rs, `unlink.filename =~ "*/conf.*"`)
```

Comment:
Does this also match /etc/configuration.txt ?

Reply:
The answer seems to be "no" according to the check in `operators.go` but I would like to confirm that my understanding is correct.

Reply:
No it shouldn't, this is not direct regular expression but patterns thus the `.` is a real dot and is escaped while parsing the pattern.

Reply:
Here the escape function https://github.com/DataDog/datadog-agent/blob/master/pkg/security/secl/eval/operators.go#L48

Reply:
Thanks.